### ISSUE TITLE
[Feature request] Abbreviated output

### ISSUE THREAD

**Issue by Road-block** [okdecm/InvasionDetector#4]
```
[Invasion Detector] Tanaris Desert START
[Invasion Detector] Eastern Plaguelands END
[Invasion Detector] Tanaris Desert ONGOING 10m:58s
[Invasion Detector] Winterspring CD 2h:59m
[Invasion Detector] Burning Steppes, Azshara NO INFO
```
When people use it to get info on guild chat, it's just a bit too much text. It scrolls the chatframe and is hard to parse.

**Comment by okdecm** [comment]
Will add some config options tonight for various things, this included.

**Comment by okdecm** [comment]
Ended up removing this functionality entirely with version 1.0.0, closing this
Appreciate it though (albeit a few years late..)
